tb-dev-challenge/tb-dev-challenge/MenuItemCell.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  MenuItemCell.swift
+//  tb-dev-challenge
+//
+//  Created by Bhavin Kapadia on 2022-02-27.
+//
+
+import UIKit
+
+class MenuItemCell: UICollectionViewCell {
+    @IBOutlet var MenuItemLabel: UILabel!
+    @IBOutlet var MenuItemPriceLabel: UILabel!
+    @IBOutlet var MenuItemImage: UIImageView!
```

Comment:
Capitalized name generally is a type, UIButton, UILabel for example, if it's an instance usually it starts with lower case, titleButton, menuItemLabel for instance

Reply:
ah okay will make that change 👍

Reply:
fixed in https://github.com/bkapadia01/HundredDaysSwift/pull/27/commits/a167ca8e3168185c4a3862128e81b076a3a95dad